### DISCUSSION TITLE
App frequently closing after update to 1.0.1

### DISCUSSION THREAD

**Discussion by gbakeman** [NeoApplications/Neo-Store#607] [Q&A]
Just after updating to version 1.0.1 recently, the app has started closing frequently. Immediately after the update it closed as expected, then it would close while trying to update other apps as well. I left it alone but noticed it was periodically sending notifications indicating it was trying to update, so I force-closed it. Now I'm trying to open it, and it closes within one or two seconds. When manually looking through logs, I can see these lines that looks interesting:

```
05-15 12:55:14.232 E/PackageManager(  958): Attempt to invoke virtual method 'boolean android.os.Bundle.getBoolean(java.lang.String, boolean)' on a null object reference
05-15 12:55:14.232 I/ActivityManager(  958): Force stopping com.machiav3lli.fdroid appid=10122 user=-1: installPackageLI
```

I also attach an extended log here. Thanks in advance for any help!

[neostore-log.txt](https://github.com/NeoApplications/Neo-Store/files/15324839/neostore-log.txt)

Edit: I tried clearing the cache only which has allowed me to start using the app, however now it is spamming me with a notification that says "Installed" and nothing else. It self-clears then adds its self back again, even if I manually clear it. Force stopping the app has halted the notifications for now.

**Comment by machiav3lli** [comment]
Thanks! Moved as issue to #610
